### PR TITLE
Fixed the diffusion profile becoming invalid when hitting the reset button (case 1269462).

### PR DESCRIPTION
**Pull request by anisunity**
When hitting the Reset button on the UI, the hash of the diffusion profile became invalid and the default values didn't make any sense. Now when hitting the button, the default values (when creating a new diffusion profile) are set and the hash is properly computed.
https://fogbugz.unity3d.com/f/cases/1269462/

Testing status
Did the steps to repro and the issue doesn't happen anymore.
Checked the hash using the debug menu and it stays the right value after the reset.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/CHANGELOG.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed SSGI compilation issues on PS4.
 - Fixed "Screen position out of view frustum" error when camera is on exactly the planar reflection probe plane.
 - Workaround issue that caused objects using eye shader to not be rendered on xbox.
+- Fixed the usage of GUIEnable for volume components (case 1280018).
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/CHANGELOG.md:86]: Leftover from another PR?

Comment (anisunity) [reply com.unity.render-pipelines.high-definition/CHANGELOG.md:86]: yeah seems like it

**Comment by anisunity** [comment]
@JulienIgnace-Unity removed

**Review by JulienIgnace-Unity — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
repro step of Anis should be enough , merging.

**Pull request merged**